src/Box/Box.cs

```diff
@@ -100,20 +102,18 @@ void OnSimulationEnded()
 		{
 			rigidBody.TopLevel = false;
 
-			rigidBody.Position = Vector3.Zero;
-			rigidBody.Rotation = Vector3.Zero;
-			rigidBody.Scale = Vector3.One;
+			rigidBody.Transform = Transform3D.Identity;
 
 			rigidBody.LinearVelocity = Vector3.Zero;
 			rigidBody.AngularVelocity = Vector3.Zero;
 
 			//Work around for #83 
 			if (enable_inital_pos)
 			{
-				GlobalPosition = initialPos;
+				GlobalTransform = initialTransform;
+				Rotation = initialRotation; // Reset to initial rotation
```

Comment:
This is redundant, an object's transform contains its rotation. 

See: https://docs.godotengine.org/en/4.3/classes/class_transform3d.html